site/guides/namespace-provisioning-ui.md

```diff
@@ -0,0 +1,184 @@
+# Namespace provisioning example using UI
+
+In this guide, we will use the package orchestration UI to:
+
+- Register blueprint and deployment repositories
```

Comment:
Suggest addling the link to tell what "blueprint", "deployment" and "repository" means here. It seems to be the term related to "repository".